.github/workflows/reusable-macos.yml

```diff
@@ -54,11 +58,14 @@ jobs:
           --config-cache \
           --with-pydebug \
           ${{ inputs.free-threading && '--disable-gil' || '' }} \
+          ${{ inputs.stackref-debug && '--with-pystackrefdebug' || '' }} \
           --prefix=/opt/python-dev \
           --with-openssl="$(brew --prefix openssl@3.0)"
     - name: Build CPython
       run: make -j8
     - name: Display build info
       run: make pythoninfo
     - name: Tests
-      run: make test
+      # Stackref debug is 3.5x slower than normal CPython,
```

Comment:
How about using the set of tests for profiling? It is about 40 tests instead of the usual ~400, so would give decent coverage without being too slow.

Reply:
Sorry how do I run those specific tests? What command with `python -m test`?